Makefile

```diff
@@ -63,8 +63,8 @@ setup-node:
 	nvm install $(NODE_VERSION) && \
 	nvm use $(NODE_VERSION)
 	@echo "📦 Installing native solc binary..."
-	@curl -L https://github.com/ethereum/solidity/releases/download/v0.8.19/solc-static-linux -o /usr/local/bin/solc && \
-	chmod +x /usr/local/bin/solc
+	@curl -L https://github.com/ethereum/solidity/releases/download/v0.8.19/solc-static-linux -o /tmp/solc && \
```

Comment:
this is to avoid installing the compiler into the shared fs (for local development - on the vm it doesn't matter). Lmk if you have a better idea